### ISSUE TITLE
Failed to Install the Xcode Editor Extension

### ISSUE THREAD

**Issue by trevinwisaksana** [schibsted/layout#123]
Hi, I have an issue with installing the **Editor Extension**. When I dragged the **Editor Extension** to my **Applications** folder, the **Extensions** panel in the **Systems Preferences does not show "Xcode Source Editor Extensions".** Moreover, Xcode is not able to detect this extension either. In other words, I believe the extension fails to install. I am using a MacBook Pro late-2013 running macOS Sierra 10.13.4 and Xcode 9.3. 

I have searched for solutions and found suggestions such as typing `sudo /usr/libexec/xpccachectl` on my Terminal (which does not work) and rebooting my MacBook Pro. **Do you have any idea on what the issue may be? Could it be a compatibility issue or am I not installing it correctly?** Thanks!

**Comment by nicklockwood** [comment]
Did you run the application after installing it?

**Comment by trevinwisaksana** [comment]
Yes, I did. It showed a short list of instructions. I opened **System Preferences** and clicked on **Extensions** as written on the instructions. But **Xcode Source Editor** is nowhere to be found.

**Comment by nicklockwood** [comment]
Hmm, I don't know what the issue could be then. Your OS version and setup sounds identical to mine.

**Comment by trevinwisaksana** [comment]
Gotcha, I'll try to install other Xcode Source Editor Extensions and see if it reacts the same way. If it does, it's probably my setup. Thanks for your help :)

**Comment by den2k** [comment]
Would there be reasons for Layout Extensions not to work with macOS 10.14 and XCODE 10?  I have successfully installed this as an XCODE Extension and even see 'Layout' in the XCODE / Editor menu but it is grayed out for some reason. Thanks.
